lib/matplotlib/mpl-data/stylelib/dark_background.mplstyle

```diff
@@ -21,3 +21,10 @@ figure.edgecolor: black
 savefig.facecolor: black
 savefig.edgecolor: black
 
+### Boxplots
+boxplot.boxprops.color: white
+boxplot.capprops.color: white
+boxplot.flierprops.color: white
+boxplot.flierprops.markeredgecolor: white
+boxplot.meanprops.markeredgecolor: white
```

Comment:
`boxplot.meanprops.markerfacecolor` and `boxplot.meanprops.markeredgecolor` are both `C2` 
Therefore, I think we do not need to redefine `boxplot.meanprops.markeredgecolor`.